hedera-node/src/main/java/com/hedera/services/txns/TransitionRunner.java

```diff
@@ -64,7 +65,8 @@
 			TokenFeeScheduleUpdate,
 			CryptoTransfer,
 			TokenDelete,
-			CryptoCreate
+			CryptoCreate,
```

Comment:
Consistence-wise - I think that we should keep refactored ops grouped by service and additionally logically grouped by the life cycle of the respective entity (create -> mutate -> delete).